backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -700,37 +670,24 @@ public static void initializeBackgroundUtilsCount() {
    */
   public static void registeredBackgroundUtilsCount() {
     int backgroundUtilsCount = 0;
-    if (System.getProperty(ModelDBConstants.BACKGROUND_UTILS_COUNT) != null) {
-      backgroundUtilsCount = getRegisteredBackgroundUtilsCount();
+    if (System.getProperty(CommonConstants.BACKGROUND_UTILS_COUNT) != null) {
+      backgroundUtilsCount = CommonUtils.getRegisteredBackgroundUtilsCount();
     }
     backgroundUtilsCount = backgroundUtilsCount + 1;
     LOGGER.trace("After registered runningBackgroundUtilsCount : {}", backgroundUtilsCount);
     System.setProperty(
-        ModelDBConstants.BACKGROUND_UTILS_COUNT, Integer.toString(backgroundUtilsCount));
+        CommonConstants.BACKGROUND_UTILS_COUNT, Integer.toString(backgroundUtilsCount));
   }
 
   public static void unregisteredBackgroundUtilsCount() {
```

Comment:
`unregisteredBackgroundUtilsCount` should be in common

Reply:
moved